REVIEW.md

```diff
@@ -0,0 +1,209 @@
+- Whenever practical, reviewers should add comments asking for a specific change as [GitHub suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-line-comments-to-a-pull-request).
+- If after discussion, an author and a reviewer still don't agree on a change and can't come up with a solution that addresses the concerns of both parties, another maintainer of the repository in question (or if not available, another Spyder core developer) should be pinged to give a second opinion and mediate the disagreement.
+
+Reviewers may give context for the review to make sure the PR author is familiar with the style and this guide. 
+
+For example,
+
+> In Spyder projects, we review pull requests by labeling which feedback needs to be addressed versus which feedback is optional. For more information, refer to the [Review Guidelines](https://github.com/spyder-ide/spyder/blob/master/REVIEW.md).
+
+## Reviewing process
+
+### For code-focused review
+
+#### Requested items
+
+The following are areas the community expects to be covered in the review of the code on each PR.
+Not all of the following individual questions will be relevant for every PR (for example, a bug fix may not need to answer the same questions as a proposal for a new feature), but these are the themes to be addressed.
+
+Review that addresses these questions on a PR will be prefixed with `[requested]` so that the PR author(s) are clear that they are blockers.
+Reviews may be tagged per comment or through the primary comment of a GitHub review.
+PR authors are responsible for addressing requested reviews for the PR to be eligible for merging.
+
+- [ ] Motivation
+    - Do we want this?
+    - Would it benefit a meaningful number of users to have this accepted into Spyder?
+    - Does it help to solve bugs?
+    - Is it a necessary refactoring?
+
+- [ ] Tests
+    - Do the tests pass in the continuous integration build?
+      If appropriate, help the contributor understand why tests failed.
+    - Are regression tests needed?
+      If a reproducible bug is being fixed in this PR, one may be needed, and reviewers should help the contributor design and implement them.
+      In case the contributor is unable to add the test, the reviewer should do so in a follow-up PR.
+
+ - [ ] Sustainability and maintenance
+    - Is the code easy to read and low on redundancy?
+    - Should variable names be improved for clarity or consistency?
+    - Should comments or docstrings be added or removed?
+    - Will the cost of maintaining a new feature be worth its benefits?
+    - Will the new code add any dependencies on other projects?
+
+- [ ] User-facing changes
+    - What use cases might this change influence?
+    - Is in-interface text (new or modified) appropriately descriptive and free of spelling or grammar issues that would make its meaning unclear?
+
```

Comment:
```suggestion

In addition to requested changes, this category also includes questions that the reviewer has for the contributor that may relate to one of these areas, or otherwise need to be answered before the PR can be merged.

```